packages/components/checkbox/src/checkbox-button.vue

```diff
@@ -8,8 +8,8 @@
       :name="name"
       :tabindex="tabindex"
       :disabled="isDisabled"
-      :true-value="trueValue ?? trueLabel"
-      :false-value="falseValue ?? falseLabel"
+      :true-value="trueValue ?? trueLabel ?? true"
```

Comment:
I feel it'd be better to extract the logic for `true-value` and `false-value` into two reusable functions. Then we can use the functions in `getLaveledValue` and [here](https://github.com/element-plus/element-plus/blob/1346bf9275a266ccf0181f373ae41cf220b5a54e/packages/components/checkbox/src/composables/use-checkbox.ts#L46).

Reply:
@dadaguai-git Can you make the adjustments as suggested above?